### PR TITLE
{Attestation} Update readme.md

### PR DESCRIPTION
**Pull request by bim-msft**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Attestation

**Review by yungezz — APPROVED** [review]
how about to add the txt file service team wants in this readm.md for future reference?

**Comment by bim-msft** [comment]
> how about to add the txt file service team wants in this readm.md for future reference?

Do you mean copying the txt content into this readme file directly?

**Comment by yungezz** [comment]
> > how about to add the txt file service team wants in this readm.md for future reference?
> 
> Do you mean copying the txt content into this readme file directly?

Yes, it's end to end sample right? Pls redact sensitive data

**Comment by bim-msft** [comment]
> > > how about to add the txt file service team wants in this readm.md for future reference?
> > 
> > 
> > Do you mean copying the txt content into this readme file directly?
> 
> Yes, it's end to end sample right? Pls redact sensitive data

Updated. These files are not sensitive.

File: src/attestation/README.md [hunk 1]
```diff
@@ -1,122 +1,74 @@
...
--n "myattestationprovider" -g "MyResourceGroup" \
---tee SgxEnclave --new-attestation-policy "newAttestationPolicyname"
-
-az attestation policy set \
--u https://myattestationprovider.eastus2.attest.azure.net \
---tee SgxEnclave --new-attestation-policy "newAttestationPolicyname"
+eyJhbGciOiJub25lIn0.eyJBdHRlc3RhdGlvblBvbGljeSI6ICJkbVZ5YzJsdmJqMGdNUzR3TzJGMWRHaHZjbWw2WVhScGIyNXlkV3hsYzN0ak9sdDBlWEJsUFQwaUpHbHpMV1JsWW5WbloyRmliR1VpWFNBOVBpQndaWEp0YVhRb0tUdDlPMmx6YzNWaGJtTmxjblZzWlhON1l6cGJkSGx3WlQwOUlpUnBjeTFrWldKMVoyZGhZbXhsSWwwZ1BUNGdhWE56ZFdVb2RIbHdaVDBpYVhNdFpHVmlkV2RuWVdKc1pTSXNJSFpoYkhWbFBXTXVkbUZzZFdVcE8yTTZXM1I1Y0dVOVBTSWtjMmQ0TFcxeWMybG5ibVZ5SWwwZ1BUNGdhWE56ZFdVb2RIbHdaVDBpYzJkNExXMXljMmxuYm1WeUlpd2dkbUZzZFdVOVl5NTJZV3gxWlNrN1l6cGJkSGx3WlQwOUlpUnpaM2d0YlhKbGJtTnNZWFpsSWwwZ1BUNGdhWE56ZFdVb2RIbHdaVDBpYzJkNExXMXlaVzVqYkdGMlpTSXNJSFpoYkhWbFBXTXVkbUZzZFdVcE8yTTZXM1I1Y0dVOVBTSWtjSEp2WkhWamRDMXBaQ0pkSUQwLUlHbHpjM1ZsS0hSNWNHVTlJbkJ5YjJSMVkzUXRhV1FpTENCMllXeDFaVDFqTG5aaGJIVmxLVHRqT2x0MGVYQmxQVDBpSkhOMmJpSmRJRDAtSUdsemMzVmxLSFI1Y0dVOUluTjJiaUlzSUhaaGJIVmxQV011ZG1Gc2RXVXBPMk02VzNSNWNHVTlQU0lrZEdWbElsMGdQVDRnYVhOemRXVW9kSGx3WlQwaWRHVmxJaXdnZG1Gc2RXVTlZeTUyWVd4MVpTazdmVHMifQ.
```
Comment (yungezz) [inline src/attestation/README.md:63]: this is fake jwt plicy right

File: src/attestation/README.md [hunk 2]
```diff
@@ -1,122 +1,74 @@
...
 ```
-az attestation signer add \
--n "myattestationprovider" -g "MyResourceGroup" \
---signer "eyAiYWxnIjoiUlMyNTYiLCAie..."
+version= 1.0;
+authorizationrules {
+[ type=="$is-debuggable", value==false ]
+&& [ type=="$product-id", value==4639 ]
+&& [ type=="$min-svn", value>= 0 ]
+&& [ type=="$sgx-mrsigner", value=="E31C9E505F37A58DE09335075FC8591254313EB20BB1A27E5443CC450B6E33E5"]
```
Comment (yungezz) [inline src/attestation/README.md:41]: this is fake value right

File: src/attestation/README.md (hunk 1 above)
Comment (bim-msft) [reply src/attestation/README.md:63]: @yungezz It's a valid policy structure but without any sensitive data.

File: src/attestation/README.md (hunk 2 above)
Comment (bim-msft) [reply src/attestation/README.md:41]: @yungezz I don't know the meaning of these fields but I think it's not a fake value. However, the data was obtained from the service team and the PM wants to use this data in the instruction script.

Comment (bim-msft) [reply src/attestation/README.md:41]: The PM also agreed that it's ok to use Github README.md to be the format of the instruction script.

**Pull request merged**